src/hooks/useCheckRevision.ts

```diff
@@ -29,7 +29,8 @@ const useCheckRevision = () => {
       newChecked.splice(checkedRevisions.indexOf(revisionIndex), 1);
     } else {
       // if there are already 4 checked revisions, print a warning
-      dispatchSetAlert('Maximum 4 Revisions', 'warning');
+      const variant: VariantType = 'warning';
+      enqueueSnackbar('Maximum 4 Revisions', { variant });
```

Comment:
The warning message can be kept in the constants file so you don't have to modify it everywhere in case it changes in the future